FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -18,7 +18,63 @@ public final class RemoteFeedLoader: FeedLoader {
 		self.client = client
 	}
 
+	private var OK_200: Int { return 200 }
+
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+			switch result {
+			case let .success((data, httpURLResponse)):
+
+				if httpURLResponse.statusCode != self.OK_200 {
+					completion(.failure(Error.invalidData))
+					return
+				}
+
+				do {
+					let value = try JSONDecoder().decode(RemoteFeedItem.self, from: data)
```

Comment:
Since we are ignoring the error being `thrown` by `JSONDecoder().decode`, there's no need to use a do/catch block, we can replace it with optional `try?`.
```suggestion
				if let value = try? JSONDecoder().decode(RemoteFeedItem.self, from: data) {
```
We can complete with `.failure(Error.invalidData)` in the `else` clause, and we will be able to combine the `statusCode` check in this `if` statement.